### PR TITLE
refactor table name design

### LINKED ISSUE #70: table name design for Flink SQL Connector
For the current implementation of flink sql nebula as sink(#57), I think the table created by flink sql is a temporary table now,  the table name is `default_catalog.default_database.table_name`, and the table name in flink sql create statement is the same as the vertex/edge name in nebula? If that's true, how to create two tables in flink sql from different nebula graph spaces with same name?

For example, in the code bellow we can see, now the table name must equals to vertex name in nebula.

- create statement: `CREATE TABLE person ...`
- nebula execution options `setTag(context.getObjectIdentifier().getObjectName())`

Shoule we add a parameter in with clause to increase flexibility?

At the same time, I noticed the `listTables` function in NebulaCatalog.java, comments are as follows: `Tag and Edge, tag starts with VERTEX. and edge starts with EDGE.`, should we be compatible with this table name design if we want to use our own catalog?

### PR DESCRIPTION
**Pull request by liuxiaocs7**
try to fix #70

### CODE REVIEW

**Comment by liuxiaocs7** [comment]
In the implementation: 
1. Use the `graph-space` and `table-name` parameters to determine the specific `tag/edge` instead of relying on the table name in Flink SQL
2. update some test logic

File: connector/src/main/java/org.apache.flink/connector/nebula/table/NebulaDynamicTableFactory.java [hunk 1]
```diff
@@ -63,6 +63,12 @@ public class NebulaDynamicTableFactory implements DynamicTableSourceFactory,
             .noDefaultValue()
             .withDescription("the nebula graph space name.");
 
+    public static final ConfigOption<String> TABLE_NAME = ConfigOptions
+            .key("table-name")
+            .stringType()
+            .noDefaultValue()
+            .withDescription("the nebula graph tag or edge name.");
+
     public static final ConfigOption<String> LABEL_NAME = ConfigOptions
```
Comment (Nicole00) [inline connector/src/main/java/org.apache.flink/connector/nebula/table/NebulaDynamicTableFactory.java:72]: what's the difference between TABLE_NAME and LABEL_NAME

File: connector/src/test/java/org/apache/flink/connector/nebula/sink/AbstractNebulaOutPutFormatITTest.java [hunk 2]
```diff
@@ -199,6 +221,7 @@ public void sinkEdgeData() throws ExecutionException, InterruptedException {
                         + PASSWORD
                         + "',"
                         + "'graph-space'='flinkSink',"
+                        + "'table-name'='friend',"
```
Comment (Nicole00) [inline connector/src/test/java/org/apache/flink/connector/nebula/sink/AbstractNebulaOutPutFormatITTest.java:224]: so the `friendTable` is the view table name in flinksql, and the `friend` is the real edge name in NebulaGraph?
if so, is it better to use `label-name` but not `table-name`?

File: connector/src/main/java/org.apache.flink/connector/nebula/table/NebulaDynamicTableFactory.java (hunk 1 above)
Comment (liuxiaocs7) [reply connector/src/main/java/org.apache.flink/connector/nebula/table/NebulaDynamicTableFactory.java:72]: Thanks for the reminder, both fields actually function the same, didn't notice it before

File: connector/src/test/java/org/apache/flink/connector/nebula/sink/AbstractNebulaOutPutFormatITTest.java (hunk 2 above)
Comment (liuxiaocs7) [reply connector/src/test/java/org/apache/flink/connector/nebula/sink/AbstractNebulaOutPutFormatITTest.java:224]: agree

**Review by Nicole00 — APPROVED** [review]
LGTM, good job!

**Pull request merged**